CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 * @jayrbolton
 
-api/* @jayrbolton @slebras
```

Comment:
Feel free to add yourself here. Mainly it auto-sets you as reviewer for PRs, and also serves as documentation for other devs

Reply:
ok, will do on the next PR.